### ISSUE TITLE
Fn::Split and Fn::Sub combination false positive

### ISSUE THREAD

**Issue by romaninsh** [martysweet/cfn-lint#221]
```
Resource: Resources > B3AdminIdentityPool > Properties > IdentityPoolName
Message: Fn::Split does not support function 'Fn::Sub' here
Documentation: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-split.html
```

code:

```
  B3AdminIdentityPool:
    Type: AWS::Cognito::IdentityPool
    Properties:
      IdentityPoolName: 
         Fn::Join: 
          - "_"
          - Fn::Split: 
            - "-"
            - !Sub ${Infra}-b3-admin
```

valid syntax replacing "-" with "_" in a string.

**Comment by martysweet** [comment]
Thanks for the issue, Fn::Sub should be added to the following list.

https://github.com/martysweet/cfn-lint/blob/c2690b1c603131ebc104a3153daab74811e54ff8/data/aws_intrinsic_functions.json#L16
